lib/minidoc/test_helpers.rb

```diff
@@ -3,11 +3,22 @@ module TestHelpers
     extend self
 
     def clear_database
-      Minidoc.database.collections.each do |coll|
-        next if coll.name.include?("system")
-        coll.remove({})
-        coll.drop_indexes
-      end
+      clear_collections
+      clear_indexes
+    end
+
+    def clear_collections
+      each_collection { |c| c.remove({}) }
+    end
+
+    def clear_indexes
+      each_collection(&:drop_indexes)
+    end
+
+    def each_collection(&block)
+      Minidoc.database.collections.
+        reject { |c| c.name.include?("system") }.
```

Comment:
Should this be stricter about where in the name `system` appears? What if we have a collection called `user_system_configurations` or something?